GVFS/GVFS.Platform.Windows/WindowsGitHooksInstaller.cs

```diff
@@ -43,7 +43,7 @@ private static string MergeHooks(GVFSContext context, string configSettingName,
             string filename;
             string[] defaultHooksLines = { };
 
-            if (configProcess.TryGetFromConfig(configSettingName, forceOutsideEnlistment: true, value: out filename))
+            if (configProcess.TryGetFromConfig(configSettingName, forceOutsideEnlistment: true, value: out filename) && filename != null)
             {
                 filename = filename.Trim(' ', '\n');
                 defaultHooksLines = File.ReadAllLines(filename);

```

Comment:
I wonder if we just had a bug here before if the config value was not set.  It looks like `File.ReadAllLines` would have  been called with an empty string.

Just mentioned this to confirm that this change in behavior is expected.

Reply:
I don't think the behavior is changed here. Previously, TryGetFromConfig would fail unless exitcode was success, meaning that a config key was present and the value was read. 

The condition changed to guard against null now that TryGetFromConfig can return null. We should figure out if that's the desired API behavior. This is the only caller of TryGetFromConfig, fwiw.

Reply:
Ahh you're right, I must have gotten the `TryGetFromConfig` implementation mixed up with one of the other helpers.